requirements.txt

```diff
@@ -31,6 +31,13 @@ django-ckeditor
 
 django-login-required-middleware==0.6.1
 
+pytest-cov
```

Comment:
Again these are not needed in the production image, we need to separate prod requirements and dev/test requirements